indico/legacy/PDFinterface/conference.py

```diff
@@ -624,11 +624,8 @@ def _processContribution(self, contrib, l):
         if self._ttPDFFormat.showContribAbstract():
             speaker_list = [self._get_speaker_name(spk) for spk in contrib.speakers]
             if speaker_list:
-                if len(speaker_list) == 1:
-                    speaker_title = i18nformat(""" _("Presenter"): """)
-                else:
-                    speaker_title = i18nformat(""" _("Presenters"): """)
-                speaker_content = speaker_title + ", ".join(speaker_list)
+                speaker_title = u' {}: '.format(ngettext('Presenter', 'Presenters', len(speaker_list)))
```

Comment:
You probably need `u''` for the two strings too - otherwise you are interpolating bytestrings (which may be non-ascii when translated) into a unicode string.